eppo/src/main/java/cloud/eppo/android/util/Utils.java

```diff
@@ -61,6 +65,16 @@ public static String logTag(Class loggingClass) {
     return logTag;
   }
 
+  public static byte[] toByteArray(InputStream input) throws IOException {
+    ByteArrayOutputStream output = new ByteArrayOutputStream();
+    byte[] buffer = new byte[BUFFER_SIZE];
+    int n;
+    while ((n = input.read(buffer)) != -1) {
+      output.write(buffer, 0, n);
+    }
+    return output.toByteArray();
+  }
```

Comment:
we were including commons-io literally just for this one method, so I removed the dep and included it here